build.gradle

```diff
@@ -10,29 +9,19 @@ buildscript {
     dependencies {
         // Add the dependency for the Google services Gradle plugin
         classpath 'com.google.gms:google-services:4.3.15'
-        //add fragment dependency
-        //def fragment_version = "1.5.5"
-        //implementation ("androidx.fragment:fragment:$fragment_version")
-
+        //classpath 'de.mannodermaus.gradle.plugins:android-junit5:1.7.1.1'
     }
 }
 
-
-
-
 plugins {
     id 'com.android.application' version '7.4.1' apply false
     id 'com.android.library' version '7.4.1' apply false
 }
 
-
-allprojects{
+allprojects {
     repositories {
         google()
         mavenCentral()
         jcenter()
-        //maven{ url "https:jitpack.io" } //for HTTP dependency
-
     }
 }
-
```

Comment:
This is a great idea but arises many concerns and errors that may not be fixed in time